blocks/news-slider/news-slider.js

```diff
@@ -161,13 +162,33 @@ export default function decorate(block) {
       link.className = 'eyebrow-link';
       link.textContent = item.textContent;
 
-      link.addEventListener('click', (event) => {
+      async function updateMainContent(articles, main) {
+        loadTemplateArticleCards(main, 'category', articles);
+      }
```

Comment:
Do we need this as a function, or can we just move `loadTemplateArticleCards()` into the link click handler?

Reply:
ah yeah, I had more in here before, so I can move it into the link handler